webapp/src/components/jira_ticket_tooltip/ticketStyle.scss

```diff
@@ -231,3 +231,22 @@
         opacity: 0.72;
     }
 }
+
+.jira-issue-tooltip-error {
```

Comment:
Add some padding

Reply:
Also get this CSS reviewed from @avas27JTG

Reply:
Hey @avas27JTG, can you please review these css